backend/src-common/src/main/java/org/eclipse/sw360/mail/MailUtil.java

```diff
@@ -113,6 +118,19 @@ public void sendMail(Set<String> recipients, String excludedRecipient, String su
         sendMail(recipients, excludedRecipient, subjectNameInPropertiesFile, textNameInPropertiesFile, notificationClass, roleName, true, textParameters);
     }
 
+    public void sendMailAsync(MimeMessage message) {
+        ExecutorService mailExecutor = Executors.newSingleThreadExecutor();
+        mailExecutor.execute(() -> {
+            try {
+                Transport.send(message);
+                log.info("Sent message successfully to user " + message.getRecipients(Message.RecipientType.TO));
```

Comment:
Since this is run asynchronously it might be good to have a more descriptive log message here and in the catch clause